cargo-culture-kit/src/lib.rs

```diff
@@ -82,14 +82,17 @@ pub mod checklist;
 pub mod exit_code;
 pub mod rules;
 
-pub use checklist::{filter_to_requested_rules_by_description,
-                    filter_to_requested_rules_from_checklist_file, find_extant_culture_file,
-                    FilterError, DEFAULT_CULTURE_CHECKLIST_FILE_NAME};
+pub use checklist::{
+    filter_to_requested_rules_by_description, filter_to_requested_rules_from_checklist_file,
+    find_extant_culture_file, FilterError, DEFAULT_CULTURE_CHECKLIST_FILE_NAME,
+};
```

Comment:
<snippy reminder to please put whitespace-only changes in their own commit, when possible>

Reply:
(this would be much easier if rustfmt worked more reliably, then we could just always run that)

Reply:
Sooth.